src/base/bittorrent/sessionimpl.cpp

```diff
@@ -320,15 +320,55 @@ struct BitTorrent::Session::ResumeSessionContext final : public QObject
 
 const int addTorrentParamsId = qRegisterMetaType<AddTorrentParams>();
 
-// Session
```

Comment:
I don't insist but I think it will help to have this comment and `// SessionImpl` to show where are the boundaries of code segments of each class.